### PR TITLE
initialize project with components

### PR DESCRIPTION
**Pull request by olanikegloria**
In this project, I learned to:
Use React Router.
Use React components.

### CODE REVIEW

File: src/components/BookList.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  ];
+
+  return (
+    <div>
+      <h1>Book List</h1>
+      <ul>
+        {books.map((book) => (
+          <li key={book.id}>
+            <h2>{book.title}</h2>
+            <p>{book.author}</p>
```
Comment (iambenkis) [inline src/components/BookList.js]: Each book  should contain `title`, and `author`  as well as a button to delete the book, kindly add the delete button on them 👍

File: src/pages/Home.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import BookList from '../components/BookList';
+
+const Home = () => (
+  <div>
+    <BookList />
```
Comment (iambenkis) [inline src/pages/Home.js]: - [ ] You should have a form to create new books in your page, I noticed you did not add it. Kindly fix that 👍

File: src/components/BookList.js (hunk 1 above)
Comment (iambenkis) [inline src/components/BookList.js]: This is one way of doing this task , however for React good practice you should create a <Book > component that takes the book data as a prop and renders the title and the auth
You could write for example

```bash
// BookItem.js
const BookItem = ({ book}) => (
  <li>
    {book.title}
    {' '}
    -
    {book.author}
    <button>
      Remove
    </button>
  </li>
);

// BookList.js
const BookList = ({ books}) => (
  <ul>
    {books.map((book) => (
      <BookItem key={book.id} book={book}  />
    ))}
  </ul>
);
```

**Review by iambenkis — CHANGES_REQUESTED** [review]
#### Hi @olanikegloria  

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Use of React routers

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/components/AddBook.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+function AddBook() {
+  return (
+    <div>
+      <form>
+        <input type='text' placeholder='Author' />
```
Comment (tufoinnkuo10) [inline src/components/AddBook.js:7]: - [optional] great job adding the input form. Please note, it gets the author and book title. You added just input for author. Kindly fix that.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @olanikegloria  

Your project is complete! Great job for making the requested changes🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request
- components well implemented
- No linter errors

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**